DelvUI/Interface/Jobs/JobHud.cs

```diff
@@ -6,7 +6,7 @@ namespace DelvUI.Interface.Jobs
 {
     public class JobHud : HudElement, IHudElementWithActor
     {
-        protected DalamudPluginInterface PluginInterface => Plugin.GetPluginInterface();
+        protected DalamudPluginInterface PluginInterface => Plugin.PluginInterface;
```

Comment:
This property might not be needed anymore. I think you replaced all the uses already.

Reply:
True, but it's there on the v4 side and is used for.... something.  I forget what.  Figure may as well keep it there to reduce the amount of difference between the two.